src/lib/wallet/MultipleAddressWallet.js

```diff
@@ -3,8 +3,6 @@ import HDKey from 'hdkey'
 import Wallet from 'ethereumjs-wallet'
 import logger from '../logger/pino-logger'
 
-const log = logger.child({ from: 'MultipleAddressWallet' })
```

Comment:
The idea of creating a `.child` per-file was to give a sort of context in the logging.

So instead of logging on every line something like:
`logger.debug('MultipleAddressWallet ', { mnemonic }, { numOfAccounts })`

You just need to:
`log.debug({ mnemonic }, {numOfAccounts })`

and it will output:
`{ from: 'MultipleAddressWallet' } { mnemonic } { numOfAccounts }`